test/stdlib/div.c

```diff
@@ -158,6 +143,27 @@ static void div_test(void** __attribute__((unused)) state)
 		mod = x % y;
 		assert_true(div == t[i].div);
 		assert_true(mod == t[i].mod);
+		if(x > y)
```

Comment:
What additional coverage do these new checks give us that checking for the correct answer does not?